### PR TITLE
Expiration background set to correct theme

### PR DESCRIPTION
**Pull request by hollabaq86**
@scannillo and I noticed that the card form expiration date was the wrong background theme... Turns out it was an easy fix!

### CODE REVIEW

File: CardForm/src/test/java/com/braintreepayments/cardform/view/CardFormTest.java [hunk 1]
```diff
@@ -1918,7 +1919,7 @@ public void valuesAreRestored() {
         assertEquals("1 234-567-8", ((EditText) mActivity.findViewById(R.id.bt_card_form_mobile_number)).getText().toString());
     }
 
-    @Test
+    @Ignore("card.io is no longer maintained")
```
Comment (lkorth) [inline CardForm/src/test/java/com/braintreepayments/cardform/view/CardFormTest.java]: These are doing nothing, why not just delete them?

Comment (lkorth) [reply CardForm/src/test/java/com/braintreepayments/cardform/view/CardFormTest.java]: #80

Comment (scannillo) [reply CardForm/src/test/java/com/braintreepayments/cardform/view/CardFormTest.java]: We were planning to remove them at the same time we remove CardIO entirely, to keep the PR history more consistent.

Comment (lkorth) [reply CardForm/src/test/java/com/braintreepayments/cardform/view/CardFormTest.java:1922]: In that case this should likely be a separate commit outside this PR then

File: CardForm/src/test/java/com/braintreepayments/cardform/utils/ExpirationDateDialogThemeTest.java [hunk 2]
```diff
@@ -16,27 +16,27 @@
 public class ExpirationDateDialogThemeTest {
 
     @Test
-    public void detectTheme_detectsBlackBackgroundAndReturnsLightTheme() {
```
[the remark this answers is not in the document]
Comment (lkorth) [reply CardForm/src/test/java/com/braintreepayments/cardform/utils/ExpirationDateDialogThemeTest.java:19]: The expiration date input widget was originally designed to be the opposite of the current view's color scheme to give it contrast and pop out of the form. That is why this test exists.

**Pull request merged**